### PR TITLE
fix tab order in qgsvectorlayer properties dialog

### PR DESCRIPTION
**Pull request by Gustry**
## Description

Fixing the focus order when we use the tab key in the vector properties dialog.

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

- [x] Commit messages are descriptive and explain the rationale for changes
- [ ] Commits which fix bugs include `fixes #11111` in the commit message next to the description
- [ ] Commits which add new features are tagged with `[FEATURE]` in the commit message
- [ ] Commits which change the UI or existing user workflows are tagged with `[needs-docs]` in the commit message and containt sufficient information in the commit message to be documented
- [x] I have read the [QGIS Coding Standards](https://docs.qgis.org/testing/en/docs/developers_guide/codingstandards.html) and this PR complies with them
- [ ] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [ ] New unit tests have been added for core changes
- [x] I have run [the `scripts/prepare-commit.sh` script](https://github.com/qgis/QGIS/blob/master/.github/CONTRIBUTE.md#contributing-to-qgis) before each commit

### CODE REVIEW

File: src/app/qgsvectorlayerproperties.cpp [hunk 1]
```diff
@@ -293,7 +293,6 @@ QgsVectorLayerProperties::QgsVectorLayerProperties(
   );
 
   QString myStyle = QgsApplication::reportStyleSheet();
-  myStyle.append( QStringLiteral( "body { margin: 10px; }\n " ) );
```
Comment (nyalldawson) [inline src/app/qgsvectorlayerproperties.cpp]: What's the rationale behind moving this?

Comment (Gustry) [inline src/app/qgsvectorlayerproperties.cpp]: I though it's better to have it centralized in the main CSS stylesheet.
I'm working on the raster dialog and I though it's better to have this CSS in one place only.

I'm happy to rollback if needed.

Comment (nyalldawson) [inline src/app/qgsvectorlayerproperties.cpp]: I think it should be rolled back - the stylesheet is used in a number of space constrained places, like the expression builder help and browser dock widget info panel, so I don't think it's a good move to always add these margins.

Comment (Gustry) [inline src/app/qgsvectorlayerproperties.cpp]: Done

**Comment by m-kuhn** [comment]
Thanks

**Pull request merged**